Source/Immutable/Private/Immutable/ImmutablePassport.cpp

```diff
@@ -184,6 +184,10 @@ void UImmutablePassport::Logout(const FImtblPassportResponseDelegate& ResponseDe
 #endif
 	if (IsStateFlagsSet(IPS_CONNECTED))
 	{
+		if (DoHardLogout)
+		{
+			SetStateFlags(IPS_HARDLOGOUT);
```

Comment:
When does this flag reset? I can only see you setting it if `DoHardLogout` is true?

Reply:
Added reseting state to logout response method.